packages/containers/galaxy/project_galaxy_containers.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+__version__ = '19.9.0.dev0'
+
+PROJECT_NAME = "galaxy-containers"
+PROJECT_OWNER = PROJECT_USERAME = "galaxyproject"
+PROJECT_URL = "https://github.com/galaxyproject/galaxy"
+PROJECT_AUTHOR = 'Galaxy Project and Community'
+PROJECT_DESCRIPTION = 'Galaxy Container Modelling and Interaction Abstractions'
```

Comment:
```suggestion
PROJECT_DESCRIPTION = 'Galaxy Container Modeling and Interaction Abstractions'
```
No preference wrt spelling, but should be consistent with packages/containers/README.rst

Reply:
Hadn't realized there was two possible spellings - xref https://www.grammarly.com/blog/modeling-or-modelling/.